src/js/entities-service/entities/actions.js

```diff
@@ -64,6 +64,12 @@ const _Model = BaseModel.extend({
         _download: attributes.download,
       });
 
+      const newFilesRelationship = this.get('_files').push({
+        id: file.id,
+        type: 'files',
+      });
+      this.set({ _files: newFilesRelationship });
```

Comment:
This sets a new `_files` relationship on the action model. Which will trigger the `modelEvent` listener on the `ActionItemView`([link](https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/views/patients/patient/flow/flow_views.js#L140)) and show/hide the paperclip icon.

Reply:
The paperclip icon showing or hiding is based on the `action.hasAttachments()` method, which checks the length of `action.get('_files')`.

So doing a `modelEvents: { 'add:comment': 'render' }` on the `ActionItemView` doesn't work.